roles/sushy_emulator/README.md

```diff
@@ -20,10 +20,10 @@ Required to installed required packages.
 * `cifmw_sushy_emulator_driver_openstack_client_config_file`: (String) Path to OpenStack config file, used by OpenStack Sushi Emulator driver. Defaults to `/etc/openstack/clouds.yaml`
 * `cifmw_sushy_emulator_driver_openstack_cloud`: (String) Cloud key reference in OpenStack config file. Defaults to `None`
 * `cifmw_sushy_emulator_hypervisor_target` (String) Hostname of the Libvirt hypervisor to connect to.
-* `cifmw_sushy_emulator_hypervisor_target_connection_ip` (String) IP Address used to connect to hypervisor, optional override for `cifmw_sushy_emulator_hypervisor_target` when specific address is required.
+* `cifmw_sushy_emulator_hypervisor_address` (String) IP Address used to connect to hypervisor, optional override for `cifmw_sushy_emulator_hypervisor_target` when specific address is required.
```

Comment:
I like this new parameter, I think we should remove cifmw_sushy_emulator_hypervisor_target. Because the hypervisor might not be managed by ansible. Now that we can set the address - pointing sushy at any hypervisor managed by ansible or not does not matter. When the hypervisor is managed, we can easily set `cifmw_sushy_emulator_hypervisor_address` from inventory/facts in the playbook calling this role.